notifier.rst

```diff
@@ -0,0 +1,562 @@
+                https://symfony.com/schema/dic/symfony/symfony-1.0.xsd">
+
+            <framework:config>
+                <framework:notifier>
+                    <framework:chatter-transport name="slack">
+                        %env(SLACK_DSN)%
+                    </framework:chatter-transport>
+                </framework:notifier>
+            </framework:config>
+        </container>
+
+    .. code-block:: php
+
+        # config/packages/notifier.php
+        $container->loadFromExtension('framework', [
+            'notifier' => [
+                'chatter_transports' => [
+                    'slack' => '%env(SLACK_DSN)%',
+                ],
+            ],
+        ]);
+
+Email Channel
+~~~~~~~~~~~~~
+
+The email channel uses the :doc:`Symfony Mailer </mailer>` to send
+notifications using the special
+:class:`Symfony\\Bridge\\TwigBridge\\Mime\\NotificationEmail`. It is
+required to install the Twig bridge along with the Inky and CSS Inliner
+Twig extensions:
+
+.. code-block:: terminal
+
+    $ composer require symfony/twig-pack twig/cssinliner-extra twig/inky-extra
+
+After this, configure the mailer. You can also set the default "from"
+email address that should be used to send the notification emails:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # config/packages/mailer.yaml
+        framework:
+            mailer:
+                dsn: '%env(MAILER_DSN)%'
+                envelope:
+                    sender: 'notifications@example.com'
+
+    .. code-block:: xml
+
+        <!-- config/packages/mailer.xml -->
+        <?xml version="1.0" encoding="UTF-8" ?>
+        <container xmlns="http://symfony.com/schema/dic/services"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xmlns:framework="http://symfony.com/schema/dic/symfony"
+            xsi:schemaLocation="http://symfony.com/schema/dic/services
+                https://symfony.com/schema/dic/services/services-1.0.xsd
+                http://symfony.com/schema/dic/symfony
+                https://symfony.com/schema/dic/symfony/symfony-1.0.xsd">
+
+            <framework:config>
+                <framework:mailer
+                    dsn="%env(MAILER_DSN)%"
+                >
+                    <framework:envelope
+                        sender="notifications@example.com"
+                    />
+                </framework:mailer>
+            </framework:config>
+        </container>
+
+    .. code-block:: php
+
+        # config/packages/mailer.php
+        $container->loadFromExtension('framework', [
+            'mailer' => [
+                'dsn' => '%env(MAILER_DSN)%',
+                'envelope' => [
+                    'sender' => 'notifications@example.com',
+                ],
+            ],
+        ]);
+
+Configure to use Failover or Round-Robin Transports
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Besides configuring one or more seperate transports, you can also use the
```

Comment:
i used to read `separate`, is `seperate` same? (I am a non english native speaker)

Reply:
Yep, you're 100% correct (not native either). https://www.grammarly.com/blog/separate-seperate/